index.html

```diff
@@ -21,29 +21,35 @@
     <link rel="preconnect" href="https://fonts.googleapis.com">
     <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
     <link href="https://fonts.googleapis.com/css2?family=Inter:wght@400;700&display=swap" rel="stylesheet">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Inter:wght@500&display=swap" rel="stylesheet">
     <title>Portafolio</title>
 </head>
 
 <body>
     <header class="nav-bar">
-        <div>
             <a href="#">Cesar Valencia</a>
-        </div>
         <nav>
-            <!--menu--> <i class="bi bi-list menu"></i>
+             <ul class="navbardesktop">
+                <li>Portofolio</li>
+                <li>About</li>
+                <li>Contact</li>
+                <li><i class="bi bi-envelope emailsend"></i></li>
+             </ul>
+             <i class="bi bi-list menu"></i>
         </nav>
     </header>
     <section class="cover">
         <div class="content">
             <h1>Hey there,<br>I´m cesar</h1>
 
-            <div><h2> I'am a software engineer and mecatronic engineer</h2></div>
-            <p>can help you build a product,featurse or website Look
```

Comment:
- [ ] Please stick to the design template and text as much as possible. So that your design matches that of Figma, please ensure to remove the extra `and mecatronic engineer`. This can be added after the project.
- [ ] 
![image](https://user-images.githubusercontent.com/84156714/186791209-e50abfc9-0853-4d88-82b3-5627dc19f574.png)

Reply:
ok im going to delete that